playbooks/roles/ocp-upgrade/README.md

```diff
@@ -1,7 +1,7 @@
 ocp-upgrade: Upgrade OCP cluster
 =========
 
-This module will upgrade an existing OCP cluster to a higher version.
+This module will upgrade an existing OCP cluster to a higher version or release image. If both upgrade version and upgrade image are specified then cluster will be upgraded to release image.
```

Comment:
This module will upgrade an existing OCP cluster based on `upgrade_version` or `upgrade_image`. If both `upgrade_version` and `upgrade_image` are specified then `upgrade_image` value gets preference.

Reply:
Incorporated review comments in the doc. Thanks!